releasenotes/notes/ucc_imaginary-8d7d9c3322899a34.yaml

```diff
@@ -0,0 +1,6 @@
+---
+features:
+  - |
+    Imaginary contributions to the UCC ansatz can be added along with the real contributions by 
+    setting the boolean flag 'include_imaginary' to True. A separate parameter is used for the real 
+    and the imaginary contribution coming from a single excitation 
```

Comment:
@mrossinek @declanmillar So setting imaginary here doubles up the number of parameters. How does this impact/affect the recent InitialPoint work that would set the starting point value for these?

Reply:
@mrossinek, @woodsp-ibm, @tanvipg, I've opened a PR [here](https://github.com/Qiskit/qiskit-nature/pull/455) that should patch this behavior. Let me know if you spot any issues with it.

Reply:
Thanks a lot, @declanmillar! The patch looks good, thanks for also including a unittest!
I pinged Tanvi on Slack, too, in order to hopefully get your PR merged asap :+1: